drivers/ethernet/eth_xlnx_gem.c

```diff
@@ -225,21 +217,22 @@ static void eth_xlnx_gem_iface_init(struct net_if *iface)
 	const struct device *dev = net_if_get_device(iface);
 	const struct eth_xlnx_gem_dev_cfg *dev_conf = dev->config;
 	struct eth_xlnx_gem_dev_data *dev_data = dev->data;
+#ifdef CONFIG_MDIO
```

Comment:
remove all ``#ifdef CONFIG_MDIO``, the phy could be on i2c or spi.

Reply:
done